### PR TITLE
mobile-version-skeleton

### PR DESCRIPTION
**Pull request by kidde60**
### Portfolio: setup and mobile version skeleton### .
**On this website I :**

- added header.
- added sections.
- added background images and icons.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+            </ul>
+        </div>
+        <hr>
+        <div class="icons">
+            <a href="#"> <img src="images/Vector.png" alt="Github"> </a>
+            <a href="#"> <img src="images/Icon-Linkedin.png" alt="LinkedIn"> </a>
+            <a href="#"> <img src="images/angelist-1.png" alt="Angelist"> </a>
+            <a href="#"> <img src="images/Icon-Twitter.png" alt="Twitter"> </a>
+            <a href="#"> <img src="images/Icon-Medium.png" alt="Medium"> </a>
+        </div>
```
Comment (danmainah) [inline index.html]: - [ ] Kindy use the `ul` tag to group your icons, and li per icon, as opposed to using `div` tag this is a project requirement.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @kidde60 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @kidde60, 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 

✅ Descriptive PR 👍 
✅ Nice design 🔥 
✅ Linters are passing 👏 


## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kidde60** [comment]
Thanks for the quick review

**Pull request merged**